### PR TITLE
Replace http with https

### PR DESCRIPTION
**Pull request by SplotyCode**
Proposed Changes

Related Issues
Code scans mark it as insecure. Here it really does not matter as the side upgrades to https it is still good practise

### CODE REVIEW

**Comment by jrfnl** [comment]
@Potherca As the plugin is no longer directly associated with DealerDirect, I'm thinking we may as well remove the `homepage` key from the `composer.json` file. What do you think ?

**Comment by Potherca** [comment]
I'd say either point it to the repo `https://github.com/PHPCSStandards/composer-installer` or remove it.
Either one is fine by me.

**Comment by jrfnl** [comment]
@Potherca As the `composer.json` file already contains a link to the repo in `support` -> `source`, I don't think `homepage` linking to the repo has any added benefit.

@SplotyCode Thanks for the PR, would you mind removing the `homepage` key instead ?

File: composer.json
Comment (SplotyCode) [inline composer.json:1]: [www.frenck.nl ](http://www.frenck.nl) was offline when i opened the pr and is still offline

Comment (jrfnl) [reply composer.json:1]: Hmm... I've send @frenck a message about that. If its doesn't come up again, we may want to remove that link too.

Comment (frenck) [reply composer.json:1]: It has moved to `https://frenck.dev`

Comment (frenck) [reply composer.json:1]: It is pretty much outdated as a whole; I'm a happy full-time open source developer nowadays :)

If you want, feel free to update it to: 

```json
    {
      "name": "Franck Nijhof",
      "email": "opensource@frenck.dev",
      "homepage": "https://frenck.dev",
      "role": "Open source developer"
    },
```

I'm happy to send in a PR if that is wanted instead.

../Frenck

Comment (frenck) [reply composer.json:1]: PS: I feel like there are quite a few key authors missing in this file... 🙈

**Review by jrfnl — APPROVED** [review]
Thanks for making those updates @SplotyCode ! All looks good to me now.

**Comment by jrfnl** [comment]
@Potherca Could you please have another look & merge if you're also okay with the changes as they are now ?

**Review by Potherca — APPROVED** [review]

**Pull request merged**